certora/run.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+const MAX_PARALLEL = 4;
+
+const specs = require(__dirname + '/specs.json');
+
+const fs = require('fs');
+const proc = require('child_process');
+const { PassThrough } = require('stream');
+const events = require('events');
+const limit = require('p-limit')(MAX_PARALLEL);
+
+for (const [spec, { files, contract }] of Object.entries(specs)) {
+  limit(run, files, contract, spec);
+}
```

Comment:
We'll need that soon, lets plan for it.

```suggestion
for (const [spec, { files, contract, args = [] }] of Object.entries(specs)) {
  limit(run, files, contract, spec, args);
}
```

Reply:
We need different arguments for different specs?